packages/block-editor/src/hooks/padding.js

```diff
@@ -4,44 +4,17 @@
 import { __ } from '@wordpress/i18n';
 import { Platform } from '@wordpress/element';
 import { getBlockSupport } from '@wordpress/blocks';
-import { __experimentalBoxControl as BoxControl } from '@wordpress/components';
+import {
+	__experimentalUseCustomUnits as useCustomUnits,
+	__experimentalBoxControl as BoxControl,
```

Comment:
Makes me wonder why these hooks (margin, padding) are not using the UnitControl component from the block editor components directly.